tests/phpunit/phpunit.xml.dist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit colors="false" bootstrap="bootstrap.php">
+  <testsuites>
+    <testsuite name="imbo-metadata-search-unit">
+      <directory>ImboMetadataSearchUnitTest</directory>
+    </testsuite>
+  </testsuites>
+
+  <php>
+    <!-- Memcached cache driver integration tests -->
+    <var name="MEMCACHED_HOST" value="" />
+    <var name="MEMCACHED_PORT" value="" />
+
+    <!-- AWS S3 tests -->
```

Comment:
Nope - some legacy-stuff because I copied over the file from Imbo. I haven't written a lot of phpunit config-files, so I used something to get me started :)